package.json

```diff
@@ -29,14 +29,16 @@
   },
   "dependencies": {
     "@ap.cx/hues": "^2.2.0",
-    "@wordpress/api-fetch": "^3.20.0",
-    "@wordpress/block-editor": "^5.1.3",
-    "@wordpress/blocks": "^6.24.1",
-    "@wordpress/components": "^11.1.1",
-    "@wordpress/compose": "^3.22.0",
+    "@wordpress/api-fetch": "^3.19.1",
```

Comment:
you are downgrading the npm packages, please use the once that are there in the `main` branch. Compare properly and accept the latest to keep.